lib/src/main/kotlin/org/promethist/core/Input.kt

```diff
@@ -58,6 +58,12 @@ data class Input(
     @get:JsonIgnore
     val intent get() = intents.firstOrNull() ?: error("No intent class recognized in input")
 
+    @get:JsonIgnore
+    val sentimets get() = classes.filter { it.type == Class.Type.Sentiment }
+
+    @get:JsonIgnore
+    val sentiment get() = classes.firstOrNull() ?: error("No sentiment class recognized in input")
```

Comment:
I prefer to return ?: Class(Class.Type.Sentiment, Sentiment.UNKNOWN.name) to ensure that sentiment sensitive code will work even when sentiment detection not available

Reply:
Error removed, added Class